symfony/client/src/orangehrmAdminPlugin/pages/employmentStatus/EditEmploymentStatus.vue

```diff
@@ -0,0 +1,135 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-background-container">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6">Edit Employment Status</oxd-text>
+
+      <oxd-divider />
+
+      <oxd-form @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-input-field
+              label="Employment Status Name"
+              v-model="employmentStatus.name"
+              :rules="rules.name"
+          />
+        </oxd-form-row>
+
+        <oxd-divider />
+
+        <oxd-form-actions>
+          <oxd-button
+              type="button"
+              displayType="ghost"
+              label="Cancel"
+              @click="onCancel"
+          />
+          <oxd-button
+              class="orangehrm-left-space"
+              displayType="secondary"
+              label="Update"
+              type="submit"
+          />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {navigate} from '@orangehrm/core/util/helper/navigation';
+
+export default {
+  props: {
+    employmentStatusId: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      employmentStatus: {
+        id: '',
+        name: '',
+      },
+      rules: {
+        name: [],
+      },
+      errors: [],
+    };
+  },
+  methods: {
+    onSave() {
+      // TODO: Loading
+      this.$http
```

Comment:
`$http` has since been removed from vue global properties. You need to initiate an intance of `APIService` for your page and use it for all the api requirements.
refer [this](https://github.com/orangehrm/orangehrm/blob/5.0/symfony/client/src/orangehrmAdminPlugin/pages/jobCategory/SaveJobCategory.vue#L74)

Reply:
Done